Update Logo and Hyperlink Button

shayla11 opened the pull request:
This is a short PR, that updates the logo to be a transparent logo. It also adds hyperlink references to the "buttons". They are no longer buttons literally, but still look like them. The easiest way was to hyperlink link text and make it appear as a button. The body tag was also not picking up the auto margin, so I changed it to ```*``` since it encompasses the whole page and it solved the problem.

(I tried to already take care of the client comment error that happens in the Servlet files, but I think I added extra lines to the files, let me know if those need to be deleted!)

shayla11 commented:
Thanks Shreya! PTAL @shreyabarua

astrelni commented on resume-buddy/src/main/webapp/style.css:
```diff
@@ -1,6 +1,6 @@
-body {
+* {
```
I think the reason you are seeing the margin issue when `body` is here is that margin is not a property that passes down to children, unlike `font-family`.

Is the goal to actually have every single element on the page centered? If so, I think the `margin` should go into it's own style block and things that are specifically for `body`, like the `max-width` should stay in this block for `body`.

astrelni replied:
I probably am not understanding CSS / HTML in this case. So the margin gets picked up by everything you need when using `*` or when adding a class to `body`, but not when the CSS is on the `body` tag itself?

astrelni commented:
Please revert the spacing changes in the java files as they seem unrelated to this change.

shayla11 commented:
I think I fixed the java spacing on those files, but on the RevieweeDataServlet.java file It says I take out a line, but when I add the line I thought I deleted, it saying I added one so I'm not sure If its fixed. @astrelni  PTAL

astrelni commented:
Github is showing RevieweeDataServlet.java have a new deletion right now. As I understand it, git would tell you that you've added a new line since you've already committed the deletion. Then when it gets added to the PR, the edit should go away.

shayla11 commented:
@astrelni PTAL

oydang commented:
@shayla11 

![Screen Shot 2020-06-30 at 12 27 25 PM](https://user-images.githubusercontent.com/5354836/86151708-17e27300-bacd-11ea-9ea1-0aab9466ac65.png)

If you go to `Files Changed` you'll see now there are some extra whitespaces on line 39 of RevieweeDataServlet.java. Try deleting the whitespaces and that should revert the file back to how it's supposed to be in `googleinterns:master`.

shayla11 commented:
Oh! That makes sense now, those white spaces where not showing for me in cloudshell. But I think I took care of it using the github file editor! PTAL @oydang
